db/db_universal_compaction_test.cc

```diff
@@ -1239,12 +1239,13 @@ TEST_P(DBTestUniversalCompaction, UniversalCompactionFourPaths) {
   options.num_levels = 1;
 
   std::vector<std::string> filenames;
-  env_->GetChildren(options.db_paths[1].path, &filenames);
+  env_->GetChildren(options.db_paths[1].path, &filenames)
```

Comment:
Should this be EXPECT_OK in the test?  When would this fail in the test scenarios and not be checked?

Reply:
The directory may not exist.

Reply:
So rather than ignoring the error, can we check for the case were the directory does not exist and do the skip all of hte deletes?